packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -1201,6 +1201,10 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, () => {
 		updateNodeAtIndex(nodeIndex, {
 			[updateInformation.key]: updateInformation.value,
 		});
+
+		if (updateInformation.key !== 'position') {
+			nodeMetadata.value[workflow.value.nodes[nodeIndex].name].parametersLastUpdatedAt = Date.now();
+		}
```

Comment:
Something to double check: From what I remember, the node position is used to determine which node gets executed first (from top to bottom). This might affect runData results. Please double check to see if that's the case.

Reply:
we discussed it already, and it's almost an edge case specially for partial executions, according to David and @despairblue.
users migth not understand why moving a node marks it as dirty.

but if we see any complains from the users we can easily remove that and just mark it as dirty if anything changes

Reply:
Ok, thanks for checking!